pandas/tests/io/parser/test_network.py

```diff
@@ -50,151 +93,142 @@ def check_compressed_urls(salaries_table, compression, extension, mode,
     tm.assert_frame_equal(url_table, salaries_table)
 
 
-class TestS3(object):
```

Comment:
That's fine. We may want to add something to the contributing docs then to ward people like me from changing existing tests to the 'more functional style' then.

https://pandas.pydata.org/pandas-docs/stable/contributing.html#transitioning-to-pytest
> Going forward, we are moving to a more functional style using the pytest framework, which offers a richer testing framework that will facilitate testing and developing. Thus, instead of writing test classes, we will write test functions like this...